### ISSUE TITLE
Make new release

### ISSUE THREAD

**Issue by pcr910303** [2ndalpha/gasmask#167]
As there were fixes and added features like #147, #148 and #155, I would like if a new release can be made.

**Comment by nucleare2** [comment]
Even without a new release, please provide appropriate build instructions.

**Comment by MuntashirAkon** [comment]
> Even without a new release, please provide appropriate build instructions.

Install CocoaPod, then `cd` to the project directory & run `pod install`, and follow the instruction given in the output.

**Comment by MuntashirAkon** [comment]
Here's a release that I've built from source: [Gas Mask.app.zip](https://github.com/2ndalpha/gasmask/files/3801579/Gas.Mask.app.zip)

**Comment by jprokos** [comment]
I tried to build using Xcode from the latest branch but it failed. What am I doing incorrectly?

```
Gas Mask Group
Target Integrity Group
/Users/admin/Downloads/gasmask-master/Gas Mask.xcodeproj The file “Pods-Gas Mask.debug.xcconfig” couldn’t be opened because there is no such file. (/Users/admin/Downloads/gasmask-master/Pods/Target Support Files/Pods-Gas Mask/Pods-Gas Mask.debug.xcconfig)
/Users/admin/Downloads/gasmask-master/Gas Mask.xcodeproj The file “Pods-Gas Mask.release.xcconfig” couldn’t be opened because there is no such file. (/Users/admin/Downloads/gasmask-master/Pods/Target Support Files/Pods-Gas Mask/Pods-Gas Mask.release.xcconfig)
Error Group
/Users/admin/Downloads/gasmask-master/Pods/Target Support Files/Pods-Gas Mask/Pods-Gas Mask.debug.xcconfig: unable to open file (in target "Gas Mask" in project "Gas Mask")
There are no accounts registered with Xcode. Add your developer account to Xcode
No signing certificate "Mac Development" found: No "Mac Development" signing certificate matching team ID "P92T7G45SR" with a private key was found.
Launcher Group
Error Group
There are no accounts registered with Xcode. Add your developer account to Xcode
No signing certificate "Mac Development" found: No "Mac Development" signing certificate matching team ID "P92T7G45SR" with a private key was found.
```

**Comment by pcr910303** [comment]
@jprokos You should probably have a (free) Apple developer account for building.

**Comment by pcr910303** [comment]
OK, so I don't use GasMask anymore so I can't really tell whether the fixed issues still persist, but with the [0.8.6 release](https://github.com/2ndalpha/gasmask/releases/tag/0.8.6) I'm guessing this is resolved.
